onnx/version_converter/adapters/scatterelements_15_16.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Adapter for ScatterElements in default domain from version 15 to 16
+
+#pragma once
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+struct ScatterElements_15_16 final: public Adapter {
```

Comment:
This looks fine. But I wonder if we need it. Isn't this a compatible extension? Wouldn't a "CompatibleAdapter" work? If no value is specified for the reduction attribute, the default value "none" produces the old behavior, right?

Reply:
Added compatible adapters instead